runtime-transaction/src/runtime_transaction.rs

```diff
@@ -0,0 +1,128 @@
+use {
+    crate::transaction_meta::{DynamicMeta, StaticMeta, TransactionMeta},
+    solana_sdk::{
+        hash::Hash,
+        message::{AddressLoader, SanitizedMessage, SanitizedVersionedMessage},
+        signature::Signature,
+        simple_vote_transaction_checker::is_simple_vote_transaction,
+        transaction::{Result, SanitizedVersionedTransaction},
+    },
+};
+
+/// RuntimeTransaction is `runtime` facing representation of transaction, while
```

Comment:
I think this comment is attached to the `RuntimeTransactionStatic`, but should probably be a module level comment?

Reply:
in 5b0c1d6a015498826e13eb2312fccd3ca8f43bea